lightstep-tracer-jre-bundle/src/main/resources/META-INF/services/com.lightstep.tracer.shared.CollectorClientProvider

```diff
@@ -0,0 +1,2 @@
+com.lightstep.tracer.shared.HttpCollectorClientProvider
+com.lightstep.tracer.shared.GrpcCollectorClientProvider
```

Comment:
it's needed so service loader will load two implementation and then choose one based on configuration from user.
without it only one implementation will be loaded from tracer-okhttp or tracer-grpc.